package.json

```diff
@@ -78,6 +78,7 @@
     "sinon": "^1.15.4",
     "sinon-chai": "^2.8.0",
     "transfer-webpack-plugin": "^0.1.4",
+    "warning": "^2.1.0",
```

Comment:
Ok, but currently there is no `dependencies` section in the `package.json` at all, things like `react` and `react-dom` are also specified as `devDependencies`.